src/frontend/src/tests/lib/components/convert/ConvertAmountExchange.spec.ts

```diff
@@ -0,0 +1,67 @@
+import ConvertAmountExchange from '$lib/components/convert/ConvertAmountExchange.svelte';
+import { formatUSD } from '$lib/utils/format.utils';
+import { assertNonNullish } from '@dfinity/utils';
+import { render } from '@testing-library/svelte';
+
+describe('ConvertAmountExchange', () => {
+	const amount = 20.25;
+	const exchangeRate = 0.01;
+
+	const props = {
+		amount,
+		exchangeRate
+	};
+
+	const divSelector = 'div[data-tid="convert-amount-exchange"]';
+	const skeletonSelector = 'div[data-tid="convert-amount-exchange-skeleton"]';
+
+	it('should display value with tilde sign if amount is provided', () => {
+		const { container, getByText } = render(ConvertAmountExchange, { props });
+
+		const div: HTMLDivElement | null = container.querySelector(divSelector);
```

Comment:
You can maybe simplify and use `getByTestId`? Or you explicitely want to use the container to retrieve the elements?

Reply:
Let's make them simpler indeed. Updated.